prototype/connection/testConnection.js

```diff
@@ -43,13 +43,13 @@ describe('connection tests >', () => {
    * returned string is the expected url
    */
   /*
-   * describe('prepareUrl >', () => {
-   *   it('prepareUrl', () => {
-   *     const url = tu.prepareUrl(ctx);
-   *     expect(url).to.have.protocol('https');
-   *   });
-   * });
-   */
+  describe('prepareUrl >', () => {
+    it('prepareUrl', () => {
+      const url = tu.prepareUrl(ctx);
+      expect(url).to.have.protocol('https');
+    });
+  });
+  */
```

Comment:
Why did you remove the asterisk in the multiline comment?

Reply:
I changed it for consistency with the other files, which I changed to multiline comments before I realized this file already had it. I think either way is fine, but I prefer it without asterisks because it keeps the indentation the same whether it's commented or not.